### DISCUSSION TITLE
mb_substr(): Argument #1 ($string) must be of type string, Closure given

### DISCUSSION THREAD

**Discussion by mallemagassa** [filamentphp/filament#5082] [Help] [answered]
Hello, at the level of the default view the name of the class which displayed instead, I would like to display the name_school field of my model the code that I use to display:

protected function getTitle(): string
    {
        if (filled(static::$title)) {
            return static::$title;
        }
        return __('filament::resources/pages/view-record.title', [
            'label' => fn (Model $record): string => $record->name_school
        ]);
    }

When viewing I get this error: mb_substr(): Argument #1 ($string) must be of type string, Closure given

**Comment by danharrin** [comment, marked as answer]
`'label' => $this->record->name_school`

**Reply by mallemagassa** [reply]
thank you very much for helping me
